docs/usage.rst

```diff
@@ -0,0 +1,719 @@
+Usage
+=====
+
+The aim of
+:mod:`audbackend`
+is to provide an
+abstract interface for
+any kind of file storage system.
+Even those,
+that have not been
+invented yet :)
+
+This tutorial is divided
+into two parts.
+In the first part,
```

Comment:
I would directly reference the parts here.

Therefore you would need to add

```rst
.. _basics:
```
before the actual sub-section and then change the code here to:

```suggestion
Under :ref:`basics`,
```